src/travis2docker/cli.py

```diff
@@ -217,13 +217,18 @@ def main(return_result=False):
         }
     else:
         os_kwargs = get_git_data(git_repo, join(root_path, 'repo'), revision)
-    if travis_yml_path:
+
+    if os_kwargs.get("variables_sh"):
+        deployv = True
+        yml_content = "deployv: True"
```

Comment:
If I understand this correctly, this adds the DeployV param whenever a `variables.sh` file is found. However, almost all projects have that file, but not all use the DeployV syntax.

I think it should b something like:

```
if variables_sh_found and not travis_yml_found:
    t2d_deployv_enabled = True
```

Reply:
I see, I have made the change.